utils/schemaValidator.js

```diff
@@ -0,0 +1,19 @@
+module.exports.valid = function (req, res, next, schema) {
+  req.checkBody(schema)
+  const errors = req.validationErrors()
+  if (errors) {
+    res.status(500).send(errors)
+    return
+  }
+  next()
+}
+
+module.exports.param = function (req, res, next, schema) {
+  req.checkParams(schema)
+  const errors = req.validationErrors()
+  if (errors) {
+    res.status(500).send(errors)
+    return
+  }
+  next()
+}
```

Comment:
you are repeating a lot of code here, the only statement that changes is the `req.checkParams` try to find a way to simplify it

Reply:
solved, thanks